src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -44,18 +44,18 @@ internal sealed class Options
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Number of bits to hash into. Must be between 1 and 31, inclusive",
                 ShortName = "bits", SortOrder = 2)]
-            public int HashBits = HashingEstimator.Defaults.HashBits;
+            public int HashBits = HashingEstimator.Defaults.NumberOfHashBits;
```

Comment:
My feeling is that `NumberOfBits` is sufficient. This is `number of bits` in a hashing module. What else can it be?

Reply:
Ok will update to `NumberOfBits`